docs/using_the_cli.md

```diff
@@ -1,21 +1,10 @@
-# Quickstart
-
-## Setup
-
-To install and use the CLI, run these commands:
-
-```
-pip install pulp-cli[pygments]
-pulp config create -e
-```
```

Comment:
It might be worthwhile to put this in the installation page under a `Quickstart` heading.

Reply:
Like TL;DR?
Would you prefer creating the config with "-e" (editor) or with "-i" (interactive)?

Reply:
> Like TL;DR?

Yes exactly.

> Would you prefer creating the config with "-e" (editor) or with "-i" (interactive)?

I prefer `-e` personally because I liked editing things with vim but I don't have a strong pref.